src/libraries/System.Net.Http.Json/ref/System.Net.Http.Json.cs

```diff
@@ -36,7 +40,7 @@ public static partial class HttpContentJsonExtensions
         public static System.Threading.Tasks.Task<T?> ReadFromJsonAsync<T>(this System.Net.Http.HttpContent content, System.Text.Json.JsonSerializerOptions? options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public static System.Threading.Tasks.Task<T?> ReadFromJsonAsync<T>(this System.Net.Http.HttpContent content, System.Text.Json.Serialization.Metadata.JsonTypeInfo<T> jsonTypeInfo, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
-    public sealed partial class JsonContent : System.Net.Http.HttpContent
+    public partial class JsonContent : System.Net.Http.HttpContent
```

Comment:
If the approach in this PR looks good, then I'll take this diff to API review (likely offline).